src/Resources/app/administration/src/module/tinect-redirects/page/tinect-redirects-index/index.js

```diff
@@ -112,12 +95,28 @@ Component.register('tinect-redirects-index', {
                 criteria.addFilter(Criteria.equals('active', this.filter.active));
             }
 
-            criteria.addFilter(Criteria.equals('hidden', this.filter.hidden));
+            if (this.filter.hidden) {
+                criteria.addFilter(Criteria.equals('active', this.filter.hidden));
+            }
```

Comment:
```suggestion
            criteria.addFilter(Criteria.equals('hidden', this.filter.hidden));
```

This change needs to be reverted. You filter on active, and this filter should always trigger, while default should not show hidden contents